components/xsd-fu/templates-java/OMEXMLModelEnumHandler.template

```diff
@@ -91,10 +91,15 @@ public class ${klass.langType}EnumHandler implements IEnumerationHandler {
 {% for value in klass.possibleValues %}\
     p.put("^\\\s*${value}\\\s*", "${value}");
 {% end %}\
-{% if klass.langType in config %}\
     // BEGIN custom enumeration mappings
```

Comment:
Perhaps an if statement around the comment here to check if klass.possibleValues is not none and greater than zero would be useful. Otherwise the comment appears in the generated code even if there are no custom enum mappings